spec/requests/api/v1/forecast_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe "Forecast API" do
+
+  it "returns the forecast for given city" do
+    WebMock.allow_net_connect!
```

Comment:
Add this to a previous test that has passed after creating fixtures and webmocks. This will allow the net connection and things will pass